### ISSUE TITLE
(Glitch) Flash indésirable après une transition Fade entre deux thèmes, quand le premier utilise des images ou émojis

### ISSUE THREAD

**Issue by wardrob-lev** [kaelhem/moiki-bug-tracking#12]
### Description

C'est un glitch visuel qui n'est pas nouveau, et qui n'est pas bloquant, mais qui crée un effet désagréable et différent du comportement habituel des transitions Fade entre les thèmes : 

- Lorsqu'un thème comporte des éléments visuels en plus du texte (image ou emojis) et que la séquence suivante utilise un thème différent, après avoir cliqué sur un choix pour changer de séquence le thème actuel disparait en fondu, mais **les éléments visuels qui ont disparu réaparaissent en flash très rapide juste avant l'affichage du thème suivant**, 

Dans l'exemple ci-dessous le thème en question comporte aussi un son sur l'action "clic sur bouton", mais j'ai constaté que le glitch se produisait ailleurs dans des conditions identiques (images dans le thème), même s'il n'y a pas de son.

### Reproduire le bug

Dans l'histoire de test, cliquer sur "ouvrir le lien" pour voir le cadre blanc de la séquence disparaître en fondu puis flasher juste avant l'apparition de la séquence suivante. Deux séquences plus loin le même phénomène se reproduit. 

L'histoire de test est là : https://moiki.fr/story/65534e6687ad6b00393d1a68
Le fichier JSON est là : https://drop.infini.fr/r/SBBQ3gH9vo#pY3+nI37cOV92/DC5nCk6rgEH65uqtqpbyMAE9SoA8M=

**Comment by kaelhem** [comment]
Je ne reproduis pas le pb sur l'histoire de test. Peut-être est-ce dû au navigateur ? Quel os / navigateur utilisez-vous ?

**Comment by wardrob-lev** [comment]
Oui, j'ai oublié de préciser le navigateur. J'ai le problème dans plusieurs versions de Firefox, avec UBUNTU 20.04

- Firefox 119.0.1
- Firefox 116.0.3
- Torbrowser 13.0.1 (basé sur Firefox 115.4.0esr)

Par contre effectivement ça ne se produit pas avec Chromium (Version 119.0.6045.123)
